ot_handler/init_pipette_for_tracking.py

```diff
@@ -0,0 +1,79 @@
+                
+                elif "multi" in pipette.name and single_tip_mode == False:
+                    if location.parent.plate_data['plate_type'] == "reservoir":
+                        location.parent.remove_volume(location.well_name, 8 * volume) ### Does the Liquid Handling class ppick up less than 8 tips???
+                    else:
+                        well_col = location.well_name[1:]
+                        wells_in_col = location.parent.columns_by_index()[well_col]
+                        wells_to_update = wells_in_col[wells_in_col.index(location):] # All wells lower than accessed well in case not row A
+
+                        if location.parent.plate_data['plate_type'] == "384-well":
+                            wells_to_update = wells_to_update[::2]
+                        
+                        for w in wells_to_update:
+                            location.parent.remove_volume(w.well_name, volume)
+                                    
+        
+        def dispense_tracked(self, volume: Optional[float] = None, location: Optional[Union[Location, TrackedWell]] = None, rate: float = 1.0, single_tip_mode=False, push_out: Optional[float] = None, ) -> InstrumentContext:
+            """Dispense liquid into a well, tracking volume if it's a TrackedWell"""
+            self.dispense(volume, location, rate, push_out=push_out)
+
+            if isinstance(location, TrackedWell):
+                if 'single' in pipette.name:
+                    location.parent.add_volume(location.well_name, volume)
+                
+                elif single_tip_mode == True:
+                    location.parent.add_volume(location.well_name, volume)
+                
+                elif "multi" in pipette.name and single_tip_mode == False:
+                    if location.parent.plate_data['plate_type'] == "reservoir":
+                        location.parent.add_volume(location.well_name, 8 * volume) ### Does the Liquid Handling class ppick up less than 8 tips???
+                    else:
+                        well_col = location.well_name[1:]
+                        wells_in_col = location.parent.columns_by_index()[well_col]
+                        wells_to_update = wells_in_col[wells_in_col.index(location):] # All wells lower than accessed well in case not row A
+
+                        if location.parent.plate_data['plate_type'] == "384-well":
+                            wells_to_update = wells_to_update[::2]
+                        
+                        for w in wells_to_update:
+                            location.parent.add_volume(w.well_name, volume)            
+            
+
+        ### To be added
+        # Support for transfer and distribute consolidate
+                
+    
+        pipette.aspirate_tracked = MethodType(aspirate_tracked, pipette)
+        pipette.dispense_tracked = MethodType(dispense_tracked, pipette)
```

Comment:
For consistency, I would suggest even overriding the native methods to avoid potential mistakes if users access the aspirate command directly, resulting in volumes falling out of sync.